docs/Cube.js-Backend/Caching.md

```diff
@@ -21,24 +20,24 @@ key is generated SQL with any existing query dependent pre-aggregations.
 
 Upon incoming request Cube.js first checks the cache using this key.
 If nothing is found it executes the query in database, returns result back alongside writing to the cache.
-Since 0.15.0 in case there's an existing value in cache it will be returned only if `refreshKey` value for query isn't changed.
+In case there's an existing value in cache it will be returned only if `refreshKey` value for query isn't changed.
```

Comment:
I guess it's quite hard to understand this sentence because it starts with a conditional part and ends with another conditional part. What if we put it like this?

```suggestion
If an existing value is present in the cache and the `refreshKey` value for the query hasn't changed, the cached value will be returned.
```

Reply:
👍